test_util/launch_configs/README.md

```diff
@@ -0,0 +1,50 @@
+# DC/OS Launch Configuration YAML
+## Design Intention
+The intention of this configuration file is to provide an interface by which
+all deployments of DC/OS, regardless of provider have a similar format, thus
+complementing the goal of dcos-launch to provide a single tool for launching
+across a variety of provider APIs
+
+## Supported Deployments and Examples
+- [Simple AWS Cloudformation](aws-cf.yaml)
+- [Zen AWS Cloudformation](aws-zen-cf.yaml)
+- [Non-DC/OS AWS Linux Cluster](aws-linux-cluster.yaml)
+- [Onprem Install on AWS Linux Cluster](aws-onprem.yaml)
+- [Onprem Install on Previously Provisioned Linux Cluster](onprem-from-info.yaml)
+- [Azure Template Deployment](azure.yaml)
+
+## Keywords and Definitions
+
+### Required for Deployment
+* `deployment_type`: Can be one of `template`, `onprem`, or `linux-cluster`.
+  * `template` refers to either an AWS cloudformation template or an Azure Resource Manager deployment template. Use of this options will require setting `deployment_name`, `template_url`, `provider`, and `provider_info`
+  * `linux_cluster` will used hard-coded templates to deploy a cluster of linux machines in a shared network on the provider of choice (currently only supports AWS). `deployment_name` and `provider` must be set. See `linux_cluster` below for options. Only `aws` provider is currently supported for this deploy option
```

Comment:
This is what would be set to deploy a plain VPC or to deploy onprem within a VPC. The differnce would be in 'deployment_type' which would be 'linux_cluster' for the plain vpc and 'onprem' in the onprem-on-vpc case